apps/vaporgui/AnimationTab.cpp

```diff
@@ -0,0 +1,45 @@
+#include "AnimationTab.h"
+#include <QVBoxLayout>
+#include "PWidgets.h"
+#include "PTimestepInput.h"
+#include "PTotalTimestepsDisplay.h"
+
+using namespace VAPoR;
+
+
+AnimationTab::AnimationTab(QWidget *parent, ControlExec *ce) : QWidget(parent), EventRouter(ce, AnimationParams::GetClassType())
+{
+    _g = new PGroup({
+        new PSection("Timestep",
+                     {
+                         new PLineItem("Current Timestep", new PTimestepInput(_controlExec)),
+                         new PTotalTimestepsDisplay(_controlExec),
+                     }),
+        new PSection("Animation",
+                     {
+                         _startTS = new PIntegerInput(AnimationParams::_startTimestepTag, "Animation Start Timestep"),
+                         _stopTS = new PIntegerInput(AnimationParams::_endTimestepTag, "Animation Last Timestep"),
+                         new PCheckbox(AnimationParams::_repeatTag, "Loop Animation Playback"),
+                         (new PIntegerInput(AnimationParams::_stepSizeTag, "Animation Play Step Size"))->SetRange(1, 10),
+                         (new PDoubleSliderEdit(AnimationParams::_maxRateTag, "Max Animation Frames Per Second"))->SetRange(1, 60),
```

Comment:
Is this a new option?  Also, shouldn't an FPS controller be in PIntSliderEdit?

Reply:
No, it was in the old GUI too. The params database stores it as a double so that is why I used the double widget.